### ISSUE TITLE
Compatibility with PAW-FC2A-K009

### ISSUE THREAD

**Issue by kukacwap** [DomiStyle/esphome-panasonic-ac#61]
I got 6 of these units and although they are Panasonic I don’t see a sing of CN-WLAN or CN-CNT or even CN-Data. I have two 4 pin CN-Modul and CN-Manual connectors and an unlabeled CN14 connector. Any idea if this works with ESP in any shape of form? I see these connectors are 12V, so this worries me.

![CB8799F4-2B43-4BCF-BBCD-5C5D754B75B1](https://user-images.githubusercontent.com/4637906/173448402-624ca32b-79ea-4c62-82d0-76c47ba2451a.jpeg)
![1E85366E-EEDC-422E-A660-C354EE099984](https://user-images.githubusercontent.com/4637906/173448418-5330a4d6-1e0b-44fd-abd6-78bbb6f48587.jpeg)

thanks

**Comment by silvioneglia** [comment]
Hello, i have the same problem. I have got two k009 and one k018. Did you find something?

**Comment by DomiStyle** [comment]
Do you happen to have the service manual available somewhere? Unfortunately without asking Panasonic or trying there's not much I can do.

You might take a look at which official wifi adapters are supported and where they connect. There's a different wifi connector used over here, so maybe they work with your unit: https://github.com/DomiStyle/esphome-panasonic-ac/issues/26

**Comment by silvioneglia** [comment]
Yes i have the manual, you can find it at this link (https://www.google.com/url?sa=t&source=web&rct=j&url=https://www.panasonicproclub.com/uploads/ES/catalogues/2021/IMM_PAW-FC_WMD_EN_04.21.pdf&ved=2ahUKEwj5jp24lc34AhXNgv0HHQ6BAfIQFnoECAgQAQ&usg=AOvVaw1MtR-YyP2OfYJS4L7zzZNI).

Unfortunatly, no information are reported about wifi external module or about pcb connectivity.
I believe the unit is build by another company and selled by panasonic (infact no panasonic logo is printed anywhere and the ir remote is a generic remote from airwell company).

For what concern the "com module" or "com manual" pins you found, i suppose that are related to a communication standard protocol (e.g. Modbus) but i do not have and info about it. If we find informations about the pins maybe a wifi module can be connected to the pcb.

**Comment by DomiStyle** [comment]
Yes, looking at the service manual it seems unlikely that these units will work with this library.
Of course, I'm more than open to adding another device if somebody is able to figure out how to control this AC.

**Comment by silvioneglia** [comment]
I have found the company who build the fan coils.
https://shop.systemair.com/en/hawair--9--ir--av/p480453
You can find there the official manual but the informations reported are more or less the same as panasonic reported.

**Comment by silvioneglia** [comment]
In the panasonic service manual info about some modbus transaction are reported. It is not clear if the fan coil has a modbus interface or if this capability is available only on the optional wall remote control. What do you think?
